demo/index.html

```diff
@@ -23,35 +23,35 @@
     <div class="vertical-section-container">
       <h1>auro-datetime</h1>
 
-      <h3>Basic date</h3>
+      <!-- <h3>Basic date</h3>
```

Comment:
Did you mean to have so many lines commented out here? Can we just remove them if they're not needed?